packages/nova-lib/lib/server/apollo_server.js

```diff
@@ -131,19 +102,54 @@ const createApolloServer = (givenOptions = {}, givenConfig = {}) => {
     options.context = deepmerge(options.context, GraphQLSchema.context);
 
     return options;
-
-  }));
+  }), 'graphqlExpressMiddleware'));
 
   // Start GraphiQL if enabled
   if (config.graphiql) {
-    graphQLServer.use(config.graphiqlPath, graphiqlExpress(_.extend(config.graphiqlOptions, {endpointURL : config.path})));
+    graphQLServer.use(config.graphiqlPath, Utils.defineName(graphiqlExpress({ ...config.graphiqlOptions, endpointURL: config.path }), 'graphiqlExpressMiddleware'));
   }
 
   // This binds the specified paths to the Express server running Apollo + GraphiQL
-  WebApp.connectHandlers.use(Meteor.bindEnvironment(graphQLServer));
+  webAppConnectHandlersUse(bindEnvironment(graphQLServer), {
+    name: 'graphQLServerMiddleware_bindEnvironment',
+    order: 30,
+  });
 };
 
-Meteor.startup(function () {
+// createApolloServer when server startup
+Meteor.startup(() => {
+  // hello
+  GraphQLSchema.addQuery(`
```

Comment:
for some reason, because i just use core packages of telescope. no posts, no comments...
when i want to create a new project with telescope core packages. the apollo server will alert a error that no query on it, so these code just ensure at least i have one query. so that i run the project created with telescope core packages will no error about this at first time.